### DISCUSSION TITLE
Timing of new release of keycloak-js

### DISCUSSION THREAD

**Discussion by gwin003** [keycloak/keycloak#12281] [Q&A] [answered]
Does the keycloak-js npm package releases follow the releases of keycloak? I was hoping to see a new version of keycloak-js published sooner rather than later, as there is a VERY small change in the nightly builds that hasn't been cut into a release yet.

Here is the change we need: https://github.com/keycloak/keycloak/commit/cdef39f76c50506681227b627e678bdbf15779a4

I'm coming from the keycloak-angular repo, where this issue was reported over there - https://github.com/mauriciovigolo/keycloak-angular/issues/395

**Comment by abstractj** [comment, marked as answer]
@gwin003 keycloak-js is released with other Keycloak artifacts, like Keycloak Server, Keycloak Operator, Keycloak Node.js. The next release should be out soon.

**Comment by gwin003** [comment]
@abstractj I noticed there was a keycloak release (18.0.1) on Jun 16, but it appears the keycloak-js artifact was not updated in 18.0.1? This change is still not present. https://github.com/keycloak/keycloak/commit/cdef39f76c50506681227b627e678bdbf15779a4

**Reply by jonkoops** [reply]
This is expected as commits are cherry-picked for release and does not mean the latest version of `main` with all the changes is included.

**Comment by abstractj** [comment]
@gwin003 that's strange because I can see the release on npmjs (https://www.npmjs.com/package/keycloak-js). Would you be able to create a GH issue with all the details? In this way we don't miss it.

**Comment by gwin003** [comment]
@abstractj I can also see it there, so thats not the issue. The issue is that the 18.0.1 release, which was published on June 16th, does not contain the fix I linked above, which was committed on May 12th. Unless I'm totally missing something, I'd expect that KeycloakInitOptions `scope?: string` property to be available to us in 18.0.1.

**Comment by gwin003** [comment]
I created an issue to track this. https://github.com/keycloak/keycloak/issues/12945

**Comment by abstractj** [comment]
Thank you @gwin003!
